### DISCUSSION TITLE
This tastes like freedom

### DISCUSSION THREAD

**Discussion by budaloops** [Stability-AI/StableStudio#35] [Ideas]
After a 10-year break from programming, I recently started again in order to find an affordable way to run Stable Diffusion on my Macbook Air M1 (8GB). I see Stable Studio as the next best step in speeding up my workflow as a logo designer on a shoe-string budget (scribble-to-image and generating mockups).

I'm a new Dad and there is a lot of pressure from my kids' mom to provide for my two adorable toddlers, and there is also a lot of pressure I set on myself to escape the rat race so I can spend more time with family (by pursuing financial freedom, instead of having my time locked up paying off two college tuitions).

Contributing to Stable Studio is the next best step I can take forward. That or continuing to acquire clients with my current 2-week-long logo design turnaround. I hope to speed this up to a 2-day turnaround using Stable Studio. And I want other creatives to have the ability to speed up their workflows too, especially creatives who tend to be drawn to macs, who more-than-likely have the regular consumer-level Macbooks with low memory. As automatic1111 contributors migrate to this project their techniques will ensure that Stable Studio is optimized for PC users so that side of it is gonna get a lot of attention. That side is taken care of.

As I learn my way around the project files in Stable Studio, and look at the code that everyone is adding to it, I look forward to discussing with all of you so that we can get further involvement in this project and support each other (maybe just having a second eye to glance over one another's work, or general moral support by seeing one another putting energy toward the same goal.

I am very grateful for the discussions we will be having here and on Discord/Twitter so that I can ask questions and learn my way around this project faster.

Can't wait to start contributing :)

Implementations I would like to supervise (on my own accord) if no one else beats me to it (and I hope they do beat me to it):

1.  [more accurate text prompt renderings](https://research.nvidia.com/labs/par/Perfusion/?ref=louisbouchard.ai)

2. [the current diffusion model creates images in a grey range, and cannot produce pure black
](https://arxiv.org/abs/2305.08891)

3. [A compression slider option for users with low GPU memory](https://arxiv.org/abs/2305.09847)

**Comment by cruhl** [comment]
That's all awesome! We'd definitely appreciate help!

**Reply by budaloops** [reply]
Thanks! I have a lot of energy to put into this

**Comment by Flashwalker** [comment]
Phah! 
Stability-AI is not something that will save you money.
Pay for every generation, naive daddy!

**Reply by budaloops** [reply]
The free version is being built right here in this GitHub repository. I could simply create images through the command line interface, but this version of Stable Diffusion here will get updated with the latest improvements
